Client/src/Pages/Status/CreateStatus/index.jsx

```diff
@@ -0,0 +1,82 @@
+import PropTypes from "prop-types";
+import { useNavigate } from "react-router";
+import { useSelector } from "react-redux";
+import { Box, Tab, useTheme } from "@mui/material";
+import TabContext from "@mui/lab/TabContext";
+import TabList from "@mui/lab/TabList";
+import GeneralSettingsPanel from "../../../Components/TabPanels/Status/GeneralSettingsPanel";
+
+/**
+ * CreateStatus page renders a page with tabs for general settings and contents.
+ * @param {string} [props.open] - Specifies the initially open tab: 'general settings' or 'content'.
+ * @returns {JSX.Element}
+ */
+
+const CreateStatus = ({ open = "general settings" }) => {
+	const theme = useTheme();
+	const navigate = useNavigate();
+	const tab = open;
+	const handleTabChange = (event, newTab) => {
+		navigate(`/status/${newTab}`);
+	};
```

Comment:
we are calling this without passing the new tab. I suggest deleting the second parameter, adding a name to the input, and getting the name from the event target, then passing it to to navigate

Reply:
yes, the onChange of TabList has the (event, value) as param/signature, so those were passed on to the handleTabChange

Reply:
@shemy-gan Please avoid resolving conversations ahead of time. 
HandleTabChange is called on line 46 without the second parameter. I am suggesting removing the second parameter altogether, an deal with that data through the name attribute from the input.

Reply:
The document/signature of onChange has event and value,

(property) onChange?: ((event: React.SyntheticEvent, value: any) => void) | undefined
Callback fired when the value changes.

so  here 	<TabList
						onChange={handleTabChange}
						aria-label="status tabs"
					>
the handleTabChange did have the second param

Reply:
@jennifer-gan I believe the point here is that if we use the name attribute we don't have to use the second parameter.

Similar to the implementation in the refactor of the various create pages I pushed the other day.

Always nice to reduce the number of parameters needed if at all possible